e2e/rbd.go

```diff
@@ -467,6 +467,18 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			// TODO: enable this test when we support rbd-nbd mounter in E2E.
+			// nbd module should be present on the host machine to run use the
+			// rbd-nbd mounter.
```

Comment:
@pkalever FYI. Once this PR is merged, and you have rebased your rbd-nd one, you can enable this test as well.